### PR TITLE
Jsl

### PR DESCRIPTION
**Pull request by yongzhi1**

### CODE REVIEW

File: sound/soc/sof/sof-pci-dev.c [hunk 1]
```diff
@@ -381,6 +381,8 @@ static const struct pci_device_id sof_pci_ids[] = {
 #if IS_ENABLED(CONFIG_SND_SOC_SOF_JASPERLAKE)
 	{ PCI_DEVICE(0x8086, 0x38c8),
 		.driver_data = (unsigned long)&jsl_desc},
+	{ PCI_DEVICE(0x8086, 0x4dc8),
+		.driver_data = (unsigned long)&jsl_desc},
```
Comment (plbossart) [inline sound/soc/sof/sof-pci-dev.c:385]: I read in other places that 0x38c8 is ICL-N and 0x4dc8 is JSL-N?
is this all JasperLake really?

Comment (yongzhi1) [reply sound/soc/sof/sof-pci-dev.c:385]: With jsl+ REV2, the id changed from 0x38c8 to 0x4dc8, but I do not know why it started with 0x38c8 at REV1 PO.

**Comment by plbossart** [comment]
can someone on the CI/QA team look into Up2 support? The topology change and the test scripts need to be updated. Thanks @fredoh9 @xiulipan @stripes416

File: sound/soc/sof/sof-pci-dev.c (hunk 1 above)
Comment (yongzhi1) [reply sound/soc/sof/sof-pci-dev.c:385]: Hi, @bardliao @lyakh @libinyang can you review the patch? thanks!!

Comment (yongzhi1) [reply sound/soc/sof/sof-pci-dev.c:385]: cc @kv2019i for the HDMI part, verified on native port and single Type-C HDMI(via Ho-HO) .

**Comment by xiulipan** [comment]
@plbossart What would be the change‘s impact? I do see some error on PCM512X
but it seems to be DSP is panic

```
kernel: [  732.049405] sof-audio-pci 0000:00:0e.0: error: trace IO error
kernel: [  732.552966] sof-audio-pci 0000:00:0e.0: error: ipc timed out for 0x60010000 size 108
kernel: [  732.552974] sof-audio-pci 0000:00:0e.0: info: preventing DSP entering D3 state to preserve context
kernel: [  732.552983] sof-audio-pci 0000:00:0e.0: status: fw entered - code 00000005
kernel: [  732.553030] sof-audio-pci 0000:00:0e.0: error: unexpected fault 0xbe00297c trace 0x00004000
kernel: [  732.553040] sof-audio-pci 0000:00:0e.0: error: hda irq intsts 0x00000000 intlctl 0xc0000001 rirb 00
kernel: [  732.553044] sof-audio-pci 0000:00:0e.0: error: dsp irq ppsts 0x00000000 adspis 0x00000000
kernel: [  732.553051] sof-audio-pci 0000:00:0e.0: error: host status 0x00000000 dsp status 0x00000000 mask 0x00000003
kernel: [  732.553055] sof-audio-pci 0000:00:0e.0: error: waking up any trace sleepers
kernel: [  732.553061] sof-audio-pci 0000:00:0e.0: error: hw params ipc failed for stream 1
```

**Comment by plbossart** [comment]
I already merged the changes for Up2 (different module name). This is different

**Pull request merged**